### PR TITLE
Add supported machine types for `tpm clear`

### PR DESCRIPTION
**Pull request by kmdkuk**
Signed-off-by: kouki <kouworld0123@gmail.com>

### CODE REVIEW

File: pkg/neco/cmd/tpm_clear.go [hunk 1]
```diff
@@ -24,13 +24,16 @@ const (
...
+	"qemu":         tpmClearLogicTypePowerOff,    // Placemat VM. Clear logic is not implemented on placemat.
+	"r640-boot-1":  tpmClearLogicTypePowerOff,    // Dell, TPM 1.2
+	"r640-boot-2":  tpmClearLogicTypeDellRedfish, // Dell, TPM 2.0
+	"r640-cs-1":    tpmClearLogicTypePowerOff,    // Dell, TPM 1.2
+	"r640-cs-2":    tpmClearLogicTypeDellRedfish, // Dell, TPM 2.0
+	"r740xd-ss-1":  tpmClearLogicTypePowerOff,    // Dell, TPM 1.2
+	"r740xd-ss-2":  tpmClearLogicTypeDellRedfish, // Dell, TPM 2.0
+	"r6525-boot-1": tpmClearLogicTypeDellRedfish, // Dell, TPM 2.0
+	"r6525-cs-1":   tpmClearLogicTypeDellRedfish, // Dell, TPM 2.0
+	"r7525-ss-1":   tpmClearLogicTypeDellRedfish, // Dell, TPM 2.0
```
Comment (ymmt2005) [inline pkg/neco/cmd/tpm_clear.go:36]: https://bozuman.cybozu.com/k/43230/

**Review by zoetrope — APPROVED** [review]
LGTM

**Review by ysksuzuki — APPROVED** [review]

File: pkg/neco/cmd/tpm_clear.go (hunk 1 above)
Comment (kmdkuk) [reply pkg/neco/cmd/tpm_clear.go:36]: @ymmt2005 
Sorry, I did not understand the intention of this comment.
Are you saying that machine type should look at the record beyond that and be careful?

Comment (ymmt2005) [reply pkg/neco/cmd/tpm_clear.go:36]: @kmdkuk 
No, it's just a comment pointing the machine type literals.

**Pull request merged**